### ISSUE TITLE
Extension refuses to activate after update to 1.9.7

### ISSUE THREAD

**Issue by tomhughes** [microsoft/vscode-cpptools#9083]
Bug type: General

**Describe the bug**
- OS and Version: Fedora 35
- VS Code Version: code-1.65.2-1646927812.el7.x86_64
- C/C++ Extension Version: 1.9.7
- Other extensions you installed (and if the issue persists after disabling them): clangd
- A clear and concise description of what the bug is.

**To Reproduce**
*Please include code sample and `task.json` files.*
1. Try and start vscode on a C project
2. Notice that debugging is not available
3. Consult the console and find:

```
mainThreadExtensionService.ts:107 Activating extension 'ms-vscode.cpptools' failed: Do not activate the extension when IntelliSense is disabled..
```

**Expected behavior**

The extension continues to start as normal.

**Comment by michelleangela** [comment]
@tomhughes 
Could you see if debugging will be activated when `C_Cpp.intelliSenseEngine` is set to true? There were recent changes to debugging that could have caused debugging to not be available when `C_Cpp.intelliSenseEngine` is set to false.

**Comment by tomhughes** [comment]
Yes that allows cpptools to load, but then the clangd extension popups a warning that it is not compatible with handing Intellisense enabled (https://github.com/clangd/vscode-clangd/blob/78cfb5de1606e3130710e0a2691686d2c69fe6e8/src/extension.ts#L38).

**Comment by FrankBlabu** [comment]
Might that be related to https://github.com/microsoft/vscode-cpptools/issues/9088 ?

**Comment by michelleangela** [comment]
#9088 is an installation issue whereas this issue is about enabling the extension's debugger features while IntelliSense server is disabled.

**Comment by junchao98** [comment]
I returned the cpptools version to 1.8.4 this problem is gone

**Comment by sean-mcmanus** [comment]
This should be fixed with 1.10.2 (pre-release).
